src/main/webapp/portal-core/js/portal/widgets/grid/plugin/RowExpanderContainer.js

```diff
@@ -77,6 +77,24 @@ Ext.define('portal.widgets.grid.plugin.RowExpanderContainer', {
     },
     
     /**
+     * Every time we are forced to regenerate a component, ensure we get a unique DOM ID to render into. 
+     * Recycling the same nodes seems to result in some difficult to nail down ext errors.
+     * 
+     * see https://jira.csiro.au/browse/AUS-2588
+     */
+    renderId: function(out, values) {
+        var idCounter = 0;
+        if (this.recordStatus[values.id]) {
+            idCounter = this.recordStatus[values.id].idCounter - 1;
```

Comment:
The ID counter is always pointing to the next free ID. The -1 happens there so we can render the "current" ID